### PR TITLE
[EA] Sequences hover over redesign

### PR DESCRIPTION
**Pull request by oetherington**
This is the redesign of sequence tooltip for the EA Forum. [Relevant Figma](https://www.figma.com/file/ZqlWFgIlVJZhpqEL4v9f1A/Q3.2-2023?node-id=525%3A4502&mode=dev).

The easiest place to test this is on the best of page:

We also have a collections tooltip which is only used in a single place; for the EA Handbook on the best of page (it's not used anywhere on LW). This was not included in the Figma, but I gave it some general styles to match the sequences tooltip:

Note that if you test this branch on the best of page you will probably run into the bug that's fixed in #8647.

┆Issue is synchronized with this [Asana task](https://app.asana.com/0/1201302964208280/1206328441628483) by [Unito](https://www.unito.io)

### CODE REVIEW

File: packages/lesswrong/components/collections/CollectionsTooltip.tsx
Comment (oetherington) [inline packages/lesswrong/components/collections/CollectionsTooltip.tsx:1]: This is the renamed `CollectionsHoverOver` component, but there are enough changes that git thinks it's a completely different file. The only place this is used is for the EA handbook on the best of page (LW don't use it at all).

File: packages/lesswrong/components/sequences/SequenceTooltip.tsx
Comment (oetherington) [inline packages/lesswrong/components/sequences/SequenceTooltip.tsx:1]: This component was completely unused, and the naming collided with the new `SequencesTooltip`.

File: packages/lesswrong/components/sequences/SequencesHoverOver.tsx
Comment (oetherington) [inline packages/lesswrong/components/sequences/SequencesHoverOver.tsx:1]: Renamed to `SequencesSummary` which is tooltip-ified in `SequencesTooltip`, to match `PostsTooltip`, `UsersTooltip` and `TagsTooltip`

**Comment by oetherington** [comment]
FYI @agnesstenlund

**Review by jpaddison3 — APPROVED** [review]
LGTM

File: packages/lesswrong/components/ea-forum/EACollectionCard.tsx
Comment (oetherington) [reply packages/lesswrong/components/ea-forum/EACollectionCard.tsx:51]: If we descructed it outside then it would need to become a dependency of the call to `useCallback`. Also, I like only having it available in the scope where it's actually used.

**Comment by oetherington** [comment]
@jpaddison3 Thanks! I'll wait until I get the OK from @agnesstenlund when she's back next week before merging this.

**Comment by agnesstenlund** [comment]
Looks really good!! Could merge as is, just two comments you may want to address before–

It's very slow to load for me, but maybe that's a local thing? (I'm in this state a long time 👇)
Screenshot 2024-01-22 at 18 41 35

Another thing: when do we cut it off? For 15 the tooltip went outside of the frame. (Maybe I even specified this in the designs and forgot 🙃)

Screenshot 2024-01-22 at 18 41 23

**Comment by oetherington** [comment]
@agnesstenlund The slowness is definitely worse on dev than on prod, but it'll still be a bit slow. Fixing it is probably out of scope of this PR though.

We don't currently cut it off anywhere - we just let it be as long as it wants. The design doesn't say anything about it, but it does only have a sequence with 8 posts. I guess we could either show a maximum number of posts (maybe with some "+x more" text?), or we could have a max height and a fade out like the posts tooltip?

**Comment by agnesstenlund** [comment]
> I guess we could either show a maximum number of posts (maybe with some "+x more" text?), or we could have a max height and a fade out like the posts tooltip?

both of these sound good, would do whatever is easiest 👍  cutting off after 8 seems good

**Pull request merged**
